pkg/Microsoft.NETCore.Platforms/runtime.json

```diff
@@ -243,10 +243,11 @@
         },
 
         "ubuntu": {
-            "#import": [ "linux" ]
+            "#import": [ "debian" ]
         },
+
         "ubuntu-x64": {
-            "#import": [ "ubuntu", "linux-x64" ]
+            "#import": [ "ubuntu", "debian-x64" ]
         },
 
         "ubuntu.14.04": {
```

Comment:
Kestrel doesn't need this since it doesn't target specific distro versioins yet. Version specific ubuntu -> debian imports might be nice to have for producing some packages, but I understand being cautious.